alicloud/resource_alicloud_ecs_network_interface_attachment_test.go

```diff
@@ -19,17 +19,106 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
 )
 
-func TestAccAliCloudECSNetworkInterfaceAttachmentBasic(t *testing.T) {
+func TestAccAliCloudEcsNetworkInterfaceAttachment_basic0(t *testing.T) {
```

Comment:
=== RUN   TestAccAliCloudEcsNetworkInterfaceAttachment_basic0
--- PASS: TestAccAliCloudEcsNetworkInterfaceAttachment_basic0 (105.50s)

=== RUN   TestAccAliCloudEcsNetworkInterfaceAttachment_basic0_twin
--- PASS: TestAccAliCloudEcsNetworkInterfaceAttachment_basic0_twin (105.67s)

=== RUN   TestAccAliCloudEcsNetworkInterfaceAttachment_basic1_twin
    provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudEcsNetworkInterfaceAttachment_basic1_twin (110.79s)

=== RUN   TestAccAliCloudECSNetworkInterfaceAttachmentMulti
--- PASS: TestAccAliCloudECSNetworkInterfaceAttachmentMulti (109.42s)
PASS